src/confdb/confdb.c

```diff
@@ -2438,3 +2442,148 @@ int confdb_certmap_to_sysdb(struct confdb_ctx *cdb,
 
     return ret;
 }
+
+/**
+ * Retrieve the list of enabled domains considering the explicit list
+ * and the 'enabled' attribute.
+ * @param cdb The database configuration context.
+ * @param ctx The memory context.
+ * @param result Output variable where the list of domains will be stored.
+ * @return 0 if the list was retrieved properly, another value on error.
+ */
+static int confdb_get_enabled_domain_list(struct confdb_ctx *cdb, TALLOC_CTX *ctx,
+                                          char ***result)
```

Comment:
Please, use _result and make it to 80  chars limit.

Reply:
Thanks! Done